### PR TITLE
Prune project links

### PR DESCRIPTION
**Pull request by bhrutledge**
After scanning through some of the projects, it seemed like most of the links to IRC and mailing lists are not reliable ways of engaging with a project. Rather than direct users to those, I thought it might be better to rely on each project's repo and documentation to point users in the right direction.

### CODE REVIEW

**Comment by souldia18** [comment]
#917

**Review by souldia18 — APPROVED** [review]

**Review by pradyunsg — DISMISSED** [review]
I'd only trim distutils-sig and #pypa/#pypa-dev IRC channels. Those other mailing lists are likely active.

**Comment by webknjaz** [comment]
I'd keep them — we have a linkcheck for detecting if something disappears.

**Comment by bhrutledge** [comment]
> I'd only trim distutils-sig and #pypa/#pypa-dev IRC channels. Those other mailing lists are likely active.

I don't think this is the case:

- https://groups.google.com/g/python-virtualenv
- https://groups.google.com/g/devpi-dev
- https://groups.google.com/g/scikit-build
- http://librelist.com/browser/bento

> we have a linkcheck for detecting if something disappears.

True, but that doesn't detect if it's still actively used.

**Comment by bhrutledge** [comment]
Based on my last comment, are y'all okay with these changes?

**Review by pradyunsg — COMMENTED** [review]
Personally, speaking for pip, packaging, packaging.python.org and distutils -- yes, this looks good to me. I imagine that this works for twine and warehouse too.

We should get an OK from the maintainers of the other projects for the corresponding changes here tho.

---

This PR is removing the IRC links and the link to distutils-sig:

/cc @jaraco for setuptools
/cc @gaborbernat for virtualenv
/cc @vsajip for distlib
/cc @cooperlees for bandersnatch
/cc @agronholm for wheel

And... IDK who to ping for devpi / scikit-build / buildout / bento.

**Comment by agronholm** [comment]
The wheel part looks good to me, nothing out of place there.

**Comment by bhrutledge** [comment]
> And... IDK who to ping for devpi / scikit-build / buildout / bento.

Me neither. Which is why I thought it would be better in general to link to consistent things like project sites and documentation, rather than changeable things like communication platforms that require maintainers to keep this page updated.

**Review by cooperlees — APPROVED** [review]
LGTM for bandersnatch.

**Review by jaraco — APPROVED** [review]

**Comment by pradyunsg** [comment]
I think it's safe to assume that the remaining folks are also OK with this. If that assumption proves wrong, reverts are cheap; so if someone has concerns, we can revert parts of this later.

File: source/key_projects.rst [hunk 1]
```diff
@@ -280,13 +259,9 @@ virtualenv
 ==========
 
 `Docs <https://virtualenv.pypa.io/en/stable/>`__ |
-`User list <http://groups.google.com/group/python-virtualenv>`__ |
-`Dev list <https://mail.python.org/mailman3/lists/distutils-sig.python.org/>`__ |
 `Issues <https://github.com/pypa/virtualenv/issues>`__ |
 `GitHub <https://github.com/pypa/virtualenv>`__ |
-`PyPI <https://pypi.org/project/virtualenv/>`__ |
-User IRC:`#pypa <https://web.libera.chat/#pypa>`__ |
```
Comment (gaborbernat) [inline source/key_projects.rst:288]: I guess we should have added Discord here...

Comment (bhrutledge) [reply source/key_projects.rst:288]: As with the other projects, I think having a link to Discord in virtualenv's docs might be sufficient.

https://github.com/pypa/virtualenv/issues/2134

**Comment by bhrutledge** [comment]
Thanks all!

**Review by souldia1 — COMMENTED** [review]
😈

**Pull request merged**